helm/flink-kubernetes-operator/conf/log4j-operator.properties

```diff
@@ -32,3 +32,5 @@ logger.conf.level = WARN
 # Avoid logging fallback key INFO messages
 logger.conf.name = org.apache.flink.configuration.Configuration
 logger.conf.level = WARN
+
+monitorInterval = 30
```

Comment:
What's the interval? Seconds? Can we add this in a comment which also explains what this does?

Reply:
I think we should disable automatic reloading by default.

Reply:
It's in seconds. Updated.